pkgs/applications/audio/tomahawk/default.nix

```diff
@@ -51,5 +51,6 @@ stdenv.mkDerivation rec {
     homepage = http://tomahawk-player.org/;
     license = licenses.gpl3Plus;
     platforms = platforms.all;
+    broken = true; # 2018-06-25
```

Comment:
How is this related to phonon-backend-vlc ?
Also, please put this in a separate commit for a clean history.

Reply:
`tomahawk` was the only other package which depends on `phonon-backend-vlc`.
However it doesn't build anymore for some time (https://hydra.nixos.org/build/71873808) and I don't think it's worth to try making it compatible with my change as it's now unmaintained for quite a while according to c76ad476b0245c474c057b451ba6fec18e7f81f1)